src/pudl/transform/eia.py

```diff
@@ -492,8 +494,7 @@ def harvesting(  # noqa: C901
                 special_case_cols[col][1],
             )
 
-        if debug:
-            col_dfs[col] = col_df
+        col_dfs[col] = col_df
```

Comment:
No reason not to just always do this, is there? Lets us keep the return type consistent.

Reply:
suuure! `col_dfs` is always a dict. but this way it will always be a {str: pd.DataFrame}. i think the only downside here is memory

Reply:
I meant that previously, the function would return a tuple with 2 items if debug was False, but 3 items if debug was True.  Now it always returns a 3 element tuple, which lets us avoid conditionals in the places where this function is getting written.

I don't *think* the memory usage is an issue, but if it were we could put the conditional back in, and just return an empty `col_dfs` when debug is False.

Reply:
Okay, I switched to only populating this dictionary of dataframes if debug is True. Otherwise it's returned but empty.